setup.py

```diff
@@ -54,24 +54,25 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     packages=setuptools.find_packages(),
-    install_requires=["jupyter_server>=1.6,<2", "h5grove==1.3.0", "h5py>=3.5"],
+    install_requires=["jupyter_server>=1.6,<3", "h5grove==1.3.0", "h5py>=3.5"],
     extras_require={"full": ["hdf5plugin"]},
-    python_requires=">=3.7",
+    python_requires=">=3.8",
     zip_safe=False,
     include_package_data=True,
     platforms="Linux, Mac OS X, Windows",
-    keywords=["Jupyter", "JupyterLab", "JupyterLab3"],
+    keywords=["Jupyter", "JupyterLab", "JupyterLab3", "JupyterLab4"],
```

Comment:
I didn't check but I guess the extension is not back-compatible with JupyterLab3, right ?

If so, I wonder if we should not drop the JLab3 `keywords` and `classifiers`

Reply:
That is a very good point and seems like folks at [Jupyter](https://github.com/jupyterlab/jupyter-renderers/blob/4d08ef19cd90d56c121587f62764ee2a135caa98/packages/fasta-extension/pyproject.toml#L13) agree with you. We should drop JupyterLab3 from keywords and classifiers